core/src/main/scala/codesearch/core/index/directory/Extractor.scala

```diff
@@ -0,0 +1,48 @@
+package codesearch.core.index.directory
+import java.io.File
+import java.nio.file.Path
+
+import org.apache.commons.io.FileUtils.{moveDirectoryToDirectory, moveFileToDirectory}
+import org.rauschig.jarchivelib.ArchiveFormat.TAR
+import org.rauschig.jarchivelib.ArchiverFactory
+import org.rauschig.jarchivelib.CompressionType.GZIP
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+private[index] trait Extractor {
+
+  /**
+    * @param from is file to unarchiving
+    * @param to is target directory
+    */
+  def unzippingMethod(from: File, to: File): Unit =
```

Comment:
If `to` is a directory, why does it say `File`? I assume it should be `Path`

Reply:
Ok